paywall/src/__tests__/unlock.js/MainWindowHandler/CachedLockState.test.ts

```diff
@@ -0,0 +1,106 @@
+import FakeWindow from '../../test-helpers/fakeWindowHelpers'
+import MainWindowHandler from '../../../unlock.js/MainWindowHandler'
+import IframeHandler from '../../../unlock.js/IframeHandler'
+import { PaywallConfig } from '../../../unlockTypes'
+
+declare const process: {
+  env: {
+    PAYWALL_URL: string
+    USER_IFRAME_URL: string
+  }
+}
+
+describe('MainWindowHandler - locked/unlocked cache', () => {
+  process.env.PAYWALL_URL = 'http://paywall'
+  process.env.USER_IFRAME_URL = 'http://app/account'
+  let fakeWindow: FakeWindow
+  const config: PaywallConfig = {
+    locks: {},
+    callToAction: {
+      default: '',
+      pending: '',
+      expired: '',
+      confirmed: '',
+    },
+  }
+
+  function getMainWindowHandler() {
+    const iframes = new IframeHandler(fakeWindow, '', '', '')
+    return new MainWindowHandler(fakeWindow, iframes, config)
+  }
+
+  describe('getCachedLockState', () => {
+    beforeEach(() => {
+      fakeWindow = new FakeWindow()
+    })
+
+    it('should return "ignore" if there is no cache', () => {
+      expect.assertions(1)
+
+      const handler = getMainWindowHandler()
+      expect(handler.getCachedLockState()).toBe('ignore')
```

Comment:
See my comment about using a constant?

Reply:
Maybe in another PR

Reply:
I don't mind using it in this, I'm going to push a comment anyways